Snowy-Path/Assets/Scripts/Character/Temperature.cs

```diff
@@ -128,7 +128,7 @@ float GetTemperatureLossRate()
         if (cloth != null)
             clothType = (float)cloth.type;
 
-        return (1 + m_weather.GetCurrentWeather().blizzardStrength) - ((1 + m_weather.GetCurrentWeather().blizzardStrength) * clothType / 100f);
+        return (1 + (float)m_weather.GetCurrentWeather().blizzardStrength / 100f) - ((1 + (float)m_weather.GetCurrentWeather().blizzardStrength / 100f) * clothType / 100f);
```

Comment:
Could be better to cache GetCurrentWeather() return value.

Reply:
Automatically cached by your Weather version